app/helpers/navigation_helper.rb

```diff
@@ -239,13 +241,11 @@ def add_sub_page_breadcrumb
 
         if @page_slug == 'home'
           empty_breadcrumbs
-        elsif @page.page_group.is_community? && @builder_landing_page.exists?
+        elsif @page_group.is_community?
           empty_breadcrumbs
-          add_landing_page_breadcrumb("/communities/#{params[:page_group_friendly_id]}")
-        elsif @builder_landing_page.exists?
+        elsif @builder_landing_page.present?
           empty_breadcrumbs
-          add_landing_page_breadcrumb("/#{params[:page_group_friendly_id]}")
-          add_sub_page_breadcrumb
```

Comment:
removing this subpage breadcrumb brings PageBuilder designs into alignment with the rest of the app. See [Figma study on breadcrumbs](https://www.figma.com/file/RD8tl0pJOkOpOip8H7ioi1/BREADCRUMB-HELL?type=design&node-id=0-1&mode=design&t=3OPTqScNQNN09zUQ-0#758159314)